include/z64actor.h

```diff
@@ -265,8 +268,7 @@ typedef struct DynaPolyActor {
     /* 0x150 */ f32 unk_150;
     /* 0x154 */ f32 unk_154;
     /* 0x158 */ s16 unk_158; // y rotation?
-    /* 0x15A */ u16 unk_15A;
-    /* 0x15C */ u32 unk_15C;
+    /* 0x15C */ u32 transformFlags;
```

Comment:
I think `transformFlags` is fine, and maybe this was suggested at some point, but what about `carriedActorFlags` just to throw out an idea?

Reply:
To keep it short:

`carriedActorFlags` sounds too much like "actor flags" to me

`carriedTransformFlags` or `transformCarriedFlags` why not but then what about the `DYNA_TRANSFORM_` flags prefix

if `transformFlags` is fine I'd rather stick with it